src/layouts/blank.vue

```diff
@@ -1,3 +1,9 @@
 <template>
   <Nuxt />
 </template>
+
+<script>
+import '~/utils/safe-focus-visible-polyfill'
```

Comment:
Is there a better way to import this into all layouts client-side only? I tried using a setting in `head` object in nuxt-config but I wasn't sure how to get the polyfill file referenced properly 🤔 Would we need to copy it out of `node_modules` during the build step or something into the static folder so it would get copied into the build output and we could reference it in a script element in the head?